### DISCUSSION TITLE
OpenAPI Integration

### DISCUSSION THREAD

**Discussion by brentvegafederal** [ngageoint/MAGE#6] [General]
Here is the OpenAPI YAML file...

See other comment.

**Comment by newmanw** [comment]
Living document resides in mage-server repository here: https://github.com/ngageoint/mage-server/blob/develop/service/src/docs/openapi.yaml

**Comment by brentvegafederal** [comment]
Here is the output of running the following command:
`openapi-generator generate -i openapi.yaml -g swift5 -o ./swift5 > output5.txt 2>&1`


[output5.txt](https://github.com/user-attachments/files/18324768/output5.txt)

**Comment by brentvegafederal** [comment]
Adding notes from several generation attempts

[openapi_generator_notes.pdf](https://github.com/user-attachments/files/18325213/openapi_generator_notes.pdf)
